### PR TITLE
git-*: add Dutch translation

### PR DESCRIPTION
**Pull request by usadson**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Since the Dutch translations only had the `git` command described, here are some translated pages of subcommands  that I use often :)

### CODE REVIEW

File: pages.nl/common/git-log.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`git log --stat`
+
+- Geef een grafiek van commits in de huidige branch:
+
+`git log --graph`
+
+- Geef de commit reeks, met enkel de eerste lijn van de commit weergegeven:
+
+`git log --oneline`
```
Comment (marchersimon) [inline pages.nl/common/git-log.md:24]: Why did you split this example in two and why did you remove the `--oneline --decorate --all --graph` example?

**Review by marchersimon — COMMENTED** [review]
I noticed that you didn't keep all the original examples. Could you please make sure not to change the order or content of any example? Thanks

File: pages.nl/common/git-log.md (hunk 1 above)
Comment (usadson) [reply pages.nl/common/git-log.md:24]: I thought it'd clarify that those are two separate options, sorry 😬

Comment (marchersimon) [reply pages.nl/common/git-log.md]: No problem. If you have suggestions for the page itself, please make a Pull Request to change the original page. All translations should be in sync with the English pages.

File: pages.nl/common/git-log.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`git log --stat`
+
+- Geef een grafiek van commits in de huidige branch, met enkel de eerste lijk van de commit weergegeven:
+
+`git log --oneline --graph`
+
+-- Geef een grafiek van alle commits, tags en branches in the complete repository:
+-
+-`git log --oneline --decorate --all --graph`
```
Comment (marchersimon) [inline pages.nl/common/git-log.md:24]:
```suggestion
- Geef een grafiek van alle commits, tags en branches in the complete repository:

`git log --oneline --decorate --all --graph`
```

**Comment by usadson** [comment]
> I noticed that you didn't keep all the original examples. Could you please make sure not to change the order or content of any example? Thanks

This is kind of new to me sorry! Should the translations be a 1:1 translation of the English page?

**Comment by usadson** [comment]
I'm so embarrassed about all these typos, should I just merge these things into a new PR?

**Comment by marchersimon** [comment]
That's no problem at all, don't worry. We automatically squash all commits before merging so there's no need to keep it clean.

**Comment by marchersimon** [comment]
@Paradact would you like to have a look at this PR, since @usadson doesn't reply anymore?
